tests/src/GC/API/GCHandle/HandleCopy.cs

```diff
@@ -38,17 +37,13 @@ public CreateObj()
             copy = handle;
         }
 
-        [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public void DestroyObj()
-        {
-            obj = null;
-        }    
-
         public bool RunTest()
         {
+            // ensuring that GC happens even with /debug mode
+            obj = null;
             GC.Collect();
+
             GC.WaitForPendingFinalizers();
```

Comment:
I don't understand the comment... we want to ensure GC happens with /debug mode by reverting the code?

Reply:
Oh I think I made an outdated commit here actually... I just reverted this test to what it was before to see the change in behavior but I didn't add the changes I made to this branch to eventually fix it. Thanks for spotting that!